### PR TITLE
Issue 919

### PR DESCRIPTION
**Pull request by krues8dr**
A start on replacing grunt, per ( opengovfoundation/madison#919 ).  #2 Will need to be merged first.

@sethetter

### CODE REVIEW

File: app/pre-build.html [hunk 1]
```diff
@@ -33,63 +33,54 @@
...
+    <script src="/lib/pagedown/Markdown.Sanitizer.js"></script>
+    <script src="/lib/pagedown-editor/Markdown.Editor.js"></script>
+    <script src="/lib/underscore/underscore-min.js"></script>
+    <script src="/lib/zeroclipboard/dist/ZeroClipboard.min.js"></script>
+    <script src="/lib/moment/min/moment.min.js"></script>
     <!-- endbuild -->
 
-    <!-- Needed for Angular File Upload IE9 shim -->
-    <script>
-      FileAPI = {
```
Comment (sethetter) [inline app/pre-build.html]: @krues8dr Is there a reason this was removed?

Comment (krues8dr) [inline app/pre-build.html]: Good catch - that was supposed to be moved up above the other assets, as in the [index file](https://github.com/opengovfoundation/madison-public/blob/issue-919/app/index.tmpl.html#L25-L30).  I'll correct that now.

Comment (krues8dr) [inline app/pre-build.html]: Hmmm.  Well this won't work at all with the build process.  This needs to be in a file, not inline.  I'll come back to this this afternoon.

Comment (sethetter) [inline app/pre-build.html]: I can tackle it real quick, no worries :)

Comment (krues8dr) [inline app/pre-build.html]: Thanks!

**Pull request merged**